### PR TITLE
Issue 3820 - update content

### PR DESCRIPTION
**Pull request by gary-tai**
Signed-off-by: Gary Tai <gary.c.tai@gmail.com>

### CODE REVIEW

File: site/content/docs/main/start-contributing.md [hunk 1]
```diff
@@ -1,20 +1,138 @@
-## Contributing
+# Start Contributing Guide
```
Comment (youngnick) [inline site/content/docs/main/start-contributing.md:1]: Very, very small nit:
The tense on this feels odd to me. Feels like it should be "New Contributor Guide", "Getting started with Contributing" or something.

**Review by youngnick — APPROVED** [review]
One tiny nit, but this looks amazing, thanks @gary-tai!

File: site/content/docs/main/start-contributing.md (hunk 1 above)
Comment (gary-tai) [reply site/content/docs/main/start-contributing.md:1]: Hi Nick,
I felt "New Contributor Guide" can be interpreted in 2 ways:
- a new guide for contributors (as in new and improved)
- a guide for new contributors

"Getting Started with Contributing" title would align with "Getting Started" title for the quickstart page.

Let me know if you'd like an edit change and I'll update.

Thanks.

Comment (OrlinVasilev) [reply site/content/docs/main/start-contributing.md:1]: I'm in favour of "Getting started with Contributing" let's be explicit !

Comment (gary-tai) [reply site/content/docs/main/start-contributing.md:1]: Sounds good. I'll make the update to "Getting Started with Contributing" title.

**Review by OrlinVasilev — APPROVED** [review]

**Comment by stevesloka** [comment]
This one has some conflicts as well @gary-tai. =)

**Comment by gary-tai** [comment]
@stevesloka - Updated. Thanks.

**Comment by stevesloka** [comment]
One quick thing @gary-tai, could you sign the commit to make the DCO bot happy?

**Comment by gary-tai** [comment]
@stevesloka - Hope I did that correctly, followed the instructions.

**Comment by stevesloka** [comment]
@gary-tai yup DCO is fine, now there are conflicts again. =(

**Comment by gary-tai** [comment]
@stevesloka , tried a few things and then I did a push -f,  don't see "resolve conflict" now. DCO looks ok. Fingers crossed.

**Review by youngnick — APPROVED** [review]
All good now, thanks @gary-tai!

**Pull request merged**